popde/density_estimate.py

```diff
@@ -362,6 +360,30 @@ class MultiDimRescalingBwKDEPy(VariableBwKDEPy):
     --------
     TBA
     """
-    % WIP
-    pass
-    
+    def __init__(self, data, input_transf=None, stdize=False, rescale=None,
+                 backend='KDEpy', bandwidth=1., dim_names=None,
+                 bandwidth_method='oned_isj')
```

Comment:
``    def __init__(self, data, input_transf=None, stdize=False, rescale=None,
                 backend='KDEpy', bandwidth=1., dim_names=None,
                 bandwidth_method='oned_isj'): 
```

Reply:
: is missing